tests/sql/settings.yaml

```diff
@@ -0,0 +1,24 @@
+$schema: ../../json_schemas/test_story.schema.yaml
+
+description: Test updating SQL settings in OS cluster.
+
+chapters:
+  - synopsis: Update SQL settings.
+    path: /_plugins/_query/settings
+    method: PUT
+    request_body:
+      payload:
+        transient: 
+          plugins:
+            sql:
+              enabled: 'true'
+              slowlog: '2'
+              cursor:
+                keep_alive: 1m
+            ppl:
+              enabled: 'true'
+            query:
+              memory_limit: '80%'
+              size_limit: '200'
```

Comment:
Are these all strings (`true`, `200`)? Logically shouldn't they be booleans/numbers?